python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1783,6 +1771,7 @@ def run(self, ccdExposure, *, dnlLUT=None, bias=None, deferredChargeCalib=None,
                 brighterFatterApplyGain = False
             else:
                 brighterFatterApplyGain = True
+                exposureMetadata["LSST ISR UNITS"] = "electron"
```

Comment:
This is a gain context in there, right?  In which case changing the units in the header doesn't seem correct.